packages/core/src/index.ts

```diff
@@ -169,6 +169,7 @@ export type {
 
 export {
   configureChains,
+  debounce,
```

Comment:
Wonder if we should have an `@wagmi/core/internal` entrypoint for utilities, like `debounce`? It's useful that we can export and use in framework-specific packages (e.g. React for `wagmi`) in the monorepo, but is it okay if folks start depending on them?

Reply:
Makes sense!